wetterdienst/core/scalar/stations.py

```diff
@@ -0,0 +1,296 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018-2021, earthobservations developers.
+# Distributed under the MIT License. See LICENSE for more info.
+import logging
+from abc import abstractmethod
+from datetime import datetime
+from enum import Enum
+from typing import List, Optional, Tuple, Union
+
+import numpy as np
+import pandas as pd
+import pytz
+
+from wetterdienst.core.scalar.core import ScalarCore
+from wetterdienst.core.scalar.result import StationsResult
+from wetterdienst.dwd.metadata.column_names import DWDMetaColumns
```

Comment:
That's one of the last remaining DWD-specific code within the `core` namespace, right?

Reply:
Upps forgot that one!